Raspberrypi packages

```diff
@@ -0,0 +1,476 @@
+
```

Comment:
This file shows various files and directories present in each main directory of raspberrypi (Generated SDK from Yocto).